features/click-to-play.json

```diff
@@ -9,7 +9,9 @@
     "exceptions": [],
     "settings": {
         "Facebook": {
-            "clicksBeforeSimpleVersion": 3
+            "clicksBeforeSimpleVersion": 3,
+            "enabled": true,
```

Comment:
Good point, Done.

Reply:
Seems like "state" should be at the root level of the feature as well, if we follow the pattern in the template (and other features).  

The question is whether it should be "enabled" here or just in the `extension-override.json`.  The latter makes more sense to me, but in this respect we are totally inconsistent... some features have "state" defined in the respective feature file, others only in the override file.  Not sure if @SlayterDev has a strong option here, otherwise I'll leave it up to you to decide which way to go.

Reply:
Good point about overrides, I  have marked this as Draft while we figure out which parts of this should live i overrides vs the feature config file.

Reply:
OK, I have now set up proper platform overrides so that Click to Load entities can be enabled/disabled per platform + are disabled by default (like features). Done.